src/plugins/wizard/public/application/utils/use/use_saved_wizard_vis.ts

```diff
@@ -58,7 +68,14 @@ export const useSavedWizardVis = (visualizationIdFromUrl: string | undefined) =>
             activeVisualization: vizStateWithoutIndex.activeVisualization,
             indexPattern: savedWizardVis.searchSourceFields.index,
           };
-          // TODO: Add validation and transformation, throw/handle errors
+
+          const isStateValid = wizardStateValidation({ styleState, visualizationState });
+          if (isStateValid) {
```

Comment:
Can this be renamed to `isInvalid`? This makes it seem like we throw an error if the state is valid